bazel/ngraph.BUILD

```diff
@@ -314,17 +271,21 @@ cc_binary(
         "-fstack-protector-all",
         '-D SHARED_LIB_PREFIX=\\"lib\\"',
         '-D SHARED_LIB_SUFFIX=\\".so\\"',
-        '-D NGRAPH_VERSION=\\"0.19.0-rc.4\\"',
+        '-D NGRAPH_VERSION=\\"0.25.1-rc.0\\"',
         "-D NGRAPH_DEX_ONLY",
         "-D NGRAPH_TBB_ENABLE",
         '-D PROJECT_ROOT_DIR=\\"\\"',
+        '-D NGRAPH_CPU_STATIC_LIB_ENABLE'
     ] + CXX_ABI,
     linkopts = [
         "-Wl,-z,noexecstack",
         "-Wl,-z,relro",
         "-Wl,-z,now",
+        "-Wl,-Bsymbolic-functions",
+        "-Wl,--exclude-libs=ALL",
```

Comment:
If the meaning/purpose of the flags are not obvious from the bazel documentation, then can you please add some code review comments what they do and what problem these flag solved. for posterity.

Reply:
These are not bazel specific flags. But a quick google search gave this:
`Bsymbolic-functions`. Binds references to all global function symbols in a program to the definitions within a user's shared library.
`--exclude-libs ALL` excludes symbols in all archive libraries from automatic export. https://linux.die.net/man/1/ld
May be we should not have this. @avijit-nervana any thoughts?